src/DependencyInjection/KnpUOAuth2ClientExtension.php

```diff
@@ -325,6 +325,8 @@ private function buildConfigurationForType(NodeDefinition $node, $type)
         $optionsNode
             ->scalarNode('client_id')->isRequired()->end()
             ->scalarNode('client_secret')->isRequired()->end()
+            ->scalarNode('client_public')->defaultNull()->end()
+            ->scalarNode('access_token')->defaultNull()->end()
```

Comment:
Instead of putting this here, add it to `OdnoklassnikiProviderConfigurator:: buildConfiguration()` - that's where you can add config options that are custom to this provider.

Reply:
removed these changes